requirements.txt

```diff
@@ -29,6 +29,12 @@ lightgbm==3.0.0
 xgboost==1.2.0
 kornia==0.4.0
 
+# Lingvo ASR dependencies
+# supported versions: (lingvo==0.6.4 with tensorflow==2.1.0)
```

Comment:
Same comment as on Dockerfile: Should we update this comment to `tensorflow-gpu` to only install a single TensorFlow?